src/igvfd/schemas/tabular_file.json

```diff
@@ -121,15 +121,39 @@
                 "peaks",
                 "prime editing guide RNA sequences",
                 "protein to protein interaction score",
-                "tissue positions",
                 "sample sort parameters",
                 "sequence barcodes",
                 "SNP effect matrix",
+                "tissue positions",
                 "variants",
                 "variant binding effects",
                 "variant effects",
                 "variant to element mapping"
             ],
+            "enum_descriptions": {
+                "barcode to element mapping": "Mapping of barcodes to genomic elements.",
+                "barcode to sample mapping": "Mapping of barcodes with to biological sample.",
+                "differential gene expression quantifications": "The quantified changes in gene expression levels between different conditions or groups.",
+                "differential transcript expression quantifications": "The quantified changes in expression levels of transcripts between conditions or groups.",
+                "editing templates": "The homology-directed DNA repair (HDR) templates containing SNVs in the library of saturation genome editing assays.",
+                "element quantifications": "The quantified levels of specific genomic elements. Formatting should follow https://github.com/IGVF-DACC/checkfiles/blob/dev/src/schemas/table_schemas/element_quant.json.",
+                "elements reference": "References for the elements studied, including details like genomic coordinates or identifiers.",
+                "fold change over control": "The ratio of expression levels or other measurements in an experimental condition relative to a control.",
+                "guide quantifications": "Information and sequencing read counts for each gRNA in a pooled CRISPR screen experiment.",
+                "guide RNA sequences": "The sequences of guide RNAs designed for genome editing. Formatting should follow https://github.com/IGVF-DACC/checkfiles/blob/dev/src/schemas/table_schemas/guide_rna_sequences.json.",
```

Comment:
... "designed for genome editing using CRISPR"

Reply:
fixed